config/default/kustomization.yaml

```diff
@@ -1,12 +1,53 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
+namePrefix: capm3-
 
-# Adds namespace to all resources.
-namespace: capm3-system
```

Comment:
Don't we need the namespace anymore? It looks like some things will end up in the `system` namespace without this (e.g. [the issuer](https://github.com/Nordix/cluster-api-provider-metal3/blob/00ec40192d4c39e6f1a98343a5c2244d15edd1b9/config/certmanager/certificate.yaml#L7)) but we still create the [capm3-system namespace](https://github.com/Nordix/cluster-api-provider-metal3/blob/feature/watch-filter-flag-furkat/config/default/namespace.yaml).

Reply:
We do need it, added, thanks.